payments/vipps/redirect.md

```diff
@@ -26,8 +26,8 @@ that the payer must confirm through the Vipps mobile app." %}
 ## Introduction
 
 * When the payer starts the purchase process, you make a `POST` request towards
-  Swedbank Pay with the collected `Purchase` information. This will generate a
-  payment with a unique `id`.
+  Swedbank Pay with the collected `Purchase` information.
+* This will generate a payment object with a unique `paymentID`.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216208)